boards/arm/stm32f4_disco/Kconfig.defconfig

```diff
@@ -10,4 +10,18 @@ if BOARD_STM32F4_DISCO
 config BOARD
 	default stm32f4_disco
 
+if USB
+
+if USB_CDC_ACM
+
+config CDC_ACM_INT_EP_ADDR
+   default 0x81
+
+config CDC_ACM_IN_EP_ADDR
+   default 0x83
+
```

Comment:
Why Endpoints addresses are 0x81/0x83 here and 0x81/0x82 in e407 ? I also suggest to add OUT endpoint config in order to configure all CDC endpoints in a consolidated way.

Reply:
@loicpoulain Thanks for reviewing! Updated EP addresses to same for both the devices and also added OUT EP address to config.